titration/utils/ui_state/user_value/buffer_ph.py

```diff
@@ -0,0 +1,31 @@
+"""
+The file for the BufferPH class
+"""
+
+from titration.utils.ui_state.user_value.user_value import UserValue
+from titration.utils import lcd_interface
+
+
+class BufferPH(UserValue):
+    """
+    This is a class for the BufferPH state of the photometer
+
+    Attributes:
+        photometer (Photometer object): the photometer is used to move through the state machine
+        previous_state (UIState object): the previous_state is used to return the last visited state
+        substate (int): the substate is used to keep track of substate of the UIState
+        string (string): the string is used to hold the user input
+    """
+
+    def save_value(self):
+        """
+        The function to save the titrator's buffer pH
+        """
+        self.titrator.buffer_ph = self.string
+
+    def loop(self):
```

Comment:
How about making a method called `get_label()` which returns a string to be used in the super().loop() method to print line 1? In the super class you can force the child to implement it similar to `save_value()`.

Reply:
This change has been implemented.